koocook_core/management/commands/_add_path.py

```diff
@@ -0,0 +1,3 @@
+def add_datatrans():
+    import sys
+    sys.path.insert(0, 'C:\\Users\\User\\PycharmProjects\\datatrans')
```

Comment:
This takes the local files and depends on a particular machine. Some changes, perhaps?

Reply:
Resolved in a24169